Release5.md

```diff
@@ -26,24 +25,56 @@ These outcomes are comprised of:
 * [Cloud-in-a-Box](https://github.com/osism/cloud-in-a-box) now comes with Swift enabled as well as the option
 for secondary NIC for external connectivity.
 
-#### Observability
-
-
-#### Systems management
-
-With the `openstack-resource-manager` a new day 2 operations tool has been added.
-Furthermore a osism role for tuned to optimize system profiles is now present.
-
-
-### Operator focused improvements
-
+### Container Management
+
+* The Kubernetes Cluster Management solution is [available as version 6.0.0](https://github.com/SovereignCloudStack/k8s-cluster-api-provider/blob/main/Release-Notes-R5.md)
+* Kubernetes v1.24 .. 1.27 are officially supported. v1.28 also works (technical preview until officially supported by capo) as do older versions (with downgrading nginx-ingress), matching OCCM and CSI versions.
+* Cluster-API (capi) v1.5.1, Cluster-API provider for Openstack (capo) v0.7.3 
+* The node images now use Ubuntu 22.04, the management host can use Ubuntu 22.04 or Debian 12.
+* Cilium v1.14.1, default now, though Calico (3.26.x) is still supported.
+* Cilium also brings the upcoming gateway API (opt-in) as technical preview.
+* The Harbor container registry can now be rollwed out with each cluster.
+* The clusters can use a registry as cache to upstream dockerhub or gcr registries.
+* The cluster management now works also on OpenStack clouds with a custom CA.
+* Storage snapshots are supported now (fix was also backported to maintained branches).
+* Diskless flavors are supported everywhere (cluster-management, health-monitor).
+* etcd defragmenetation and backup.
+* Controls for pod and service IP ranges.
+
+### Preview: Cluster-Stacks
+The old scripts that are used to create, change and delete Kubernetes clusters with
+cluster-api will be replaced by a proper Operator in the next release.
+Design document and technical preview code can be found in the
+[cluster-stacks](https://github.com/SovereignCloudStack/cluster-stacks)
+and [cluster-stack-operator](https://github.com/SovereignCloudStack/cluster-stack-operator)
+repositories. This solution will fit more nicely into the CNCF landscape and
+also allow for easier support of IaaS solutions that do not comply to our SCS
+IaaS standards.
+
+### Operations and IAM related
+
+* A number of improvements when using identity federation via OIDC has been added, including
+  addressing openstack CLI usage with PKCE Device Authz Grant, logout, and the usage of a
+  proxy realm in keycloak. Improvements have been contributed to upstream keystone.
+* With the `openstack-resource-manager` a new day 2 operations tool has been added.
+  Furthermore a osism role for tuned to optimize system profiles is now present.
```

Comment:
```suggestion
  Furthermore an osism role for tuned to optimize system profiles is now present.
```

Reply:
Fixed this one along with the other typos that you found and some that ispell found in commit https://github.com/SovereignCloudStack/release-notes/pull/9/commits/3839913eab87914241c670261d659037cf0ba025.
I should have done this last night, sorry.